src/features/people/personPageDetailsSlice.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const personPageDetails = createSlice({
+  name: "personPageDetails",
+  initialState: {
+      status: "tako",
```

Comment:
Co to za status "tako"?

Reply:
xD, nie usunalem tego, to zostalo z wersji roboczej , dobrze, ze to zauwazyles :)